src/adapters/plugin/plugin.js

```diff
@@ -178,6 +187,24 @@ class Plugin {
     return this.ipcSocket.sendJson(msg);
   }
 
+  start() {
+    // If we need embedded spaces, then consider changing to npm
+    // module called splitargs
+    let args = this.exec.split(' ');
+    this.process = spawn(args[0], args.slice(1));
+
+    this.process.stdout.on('data', data => {
+      console.log(data.toString());
+    });
+    this.process.stderr.on('data', data => {
+      console.log(data.toString());
+    });
+    this.process.on('close', code => {
+      console.log('Plugin:', this.pluginId, 'died, code =', code, 'restarting...');
+      this.start();
```

Comment:
This should probably be rate limited - so that if there is an error in the exec line it doesn't just respawn and consume 100% of the CPU